examples/emodel_pipeline_local_python/scripts/create_venv.sh

```diff
@@ -14,7 +14,7 @@ cd software
 ../$VENV/bin/pip install wheel
 
 git clone https://github.com/BlueBrain/BluePyEModel.git
-../$VENV/bin/pip install -e ./bluepyemodel
```

Comment:
You don't need to change that, I think

Reply:
when you clone it, the folder name has the uppercase letters

Reply:
oh yeah, you are right